tests/publiccloud/xfsprepare.pm

```diff
@@ -0,0 +1,79 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Prepare secondary disk for usage in xfstests
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use version_utils qw(is_sle);
+use utils;
+
+# xfstests configuration file, as required by xfstests/run
+my $INST_DIR = '/opt/xfstests';
+my $CONFIG_FILE = "$INST_DIR/local.config";
+
+# Install requirements
+sub install_xfstests {
+    # Add filesystem repository, which contains the xfstests
+    assert_script_run('zypper -n up', timeout => 300);
```

Comment:
wouldn't you like to use `zypper_call` instead, since you are already importing `utils.pm`?

Reply:
This actually should go away .....